src/js/utils/events.js

```diff
@@ -248,7 +249,11 @@ export function on(elem, type, fn) {
           if (event.isImmediatePropagationStopped()) {
             break;
           } else {
-            handlersCopy[m].call(elem, event, hash);
+            try {
```

Comment:
That would make the code a lot uglier than it already is for little to no benefit. There should *never* be so many event handlers for any single event that the performance difference becomes an issue.